off-chain/test/Examples/AdditionExample.purs

```diff
@@ -0,0 +1,54 @@
+module AdditionExample(executeExample) where
+
+import Actions (class SeathAction)
+import Contract.PlutusData (class ToData, toData)
+import Contract.Prelude (genericShow, unwrap)
+import Contract.Transaction (Transaction, TransactionHash)
+import Data.Generic.Rep (class Generic)
+import Data.Newtype (class Newtype)
+import Data.UInt (UInt)
+import Effect.Aff (Aff)
+import Prelude (Unit, class Eq, class Show)
+import SeathData (UserAction)
+import Undefined (undefined)
+
+newtype AdditionAction = AddAmount UInt
```

Comment:
The type used on-chain has changed, so we need to find the right type instead of UInt